documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -6,6 +6,7 @@
 
 * For Stateless and Rapid scans, the scanId and scan type being run are now stored in the codeLocations section of the status.json file. For a given scanId, the scan type can be DETECTOR, BINARY_SCAN, SIGNATURE_SCAN, or CONTAINER_SCAN.
 * Stateless Signature and Package Manager scans now support the <code>--detect.blackduck.rapid.compare.mode</code> flag. Values are ALL, BOM_COMPARE, or BOM_COMPARE_STRICT. See the [Stateless Scans page](runningdetect/statelessscan.md) for further details. 
+* [Component Location Analysis](properties/configuration/component-location-analysis.md) is now available for offline and Rapid/Stateless online scans of NPM, Maven, Gradle and NuGet projects.
```

Comment:
I think you already mention the supported detector types in separate page as a limitation, so that can be removed here, but it seems okay if that is the requirement.

Reply:
I see the release note as something customers will see first, so mentioning what projects we support may encourage/discourage them from bothering to read the linked page based on if it applies to them or not